### ISSUE TITLE
otto build destroy? 

### ISSUE THREAD

**Issue by mreferre** [hashicorp/otto#189]
One could tear down a cloud deploy (`otto deploy destroy`) could tear down an infra (`otto infra destroy`) but apparently there seems to be no way to clean up the builds (templates generated by `otto build`). Is there a plan to have an `otto build destroy`?

**Comment by jrnt30** [comment]
The intended functionality being the AMI would be removed from your AWS account?

**Comment by mreferre** [comment]
Yes that would be the idea. Think about someone testing/deploying multiple apps. Similarly to how you "tear down" instances that you deployed to test it, you'd expect to be able to "clean up" all the builds for that test and that you may no longer be using. Surely you can go directly to the cloud provider and purge those templates yourself but it seems to me more elegant have a "destroy" feature for those that fits into the "tear down process" provided by Otto out of the box.
